apps/rush-lib/src/scripts/install-run-rush.ts

```diff
@@ -23,8 +23,13 @@ import {
 } from './install-run';
 
 const PACKAGE_NAME: string = '@microsoft/rush';
+const RUSH_PREVIEW_VERSION: string = 'RUSH_PREVIEW_VERSION';
 
 function getRushVersion(): string {
+  if (process.env[RUSH_PREVIEW_VERSION] !== undefined) {
+    return process.env[RUSH_PREVIEW_VERSION]!;
```

Comment:
I think we should do some validation on the value we get here to make sure it looks like a semver version. Maybe just a regex?

We should also log that we got the version from the environment variable.

Reply:
There is some validation in `install-run.js`. Should we let that deal with validating version, since there wasn't any check for version in rush.json as well?

Reply:
@iclanton